packages/xrpl/src/models/transactions/index.ts

```
@@ -37,9 +37,9 @@ export { CheckCancel } from './checkCancel'
 export { CheckCash } from './checkCash'
 export { CheckCreate } from './checkCreate'
 export { Clawback } from './clawback'
-export { CredentialAccept } from './CredentialAccept'
-export { CredentialCreate } from './CredentialCreate'
-export { CredentialDelete } from './CredentialDelete'
+export { CredentialAccept } from './credentialAccept'
```

ESLint

[error] Missing file extension for "./credentialAccept"

(import/extensions)